src/registrar/views/portfolios.py

```diff
@@ -341,6 +407,91 @@ def get(self, request, pk):
             },
         )
 
+    def post(self, request, pk):
+        """
+        Handles adding and removing domains for a portfolio invitee.
+        """
+        added_domains = request.POST.get("added_domains")
+        removed_domains = request.POST.get("removed_domains")
+        portfolio_invitation = get_object_or_404(PortfolioInvitation, pk=pk)
+        email = portfolio_invitation.email
+
+        added_domain_ids = self._parse_domain_ids(added_domains, "added domains")
+        if added_domain_ids is None:
+            return redirect(reverse("invitedmember-domains", kwargs={"pk": pk}))
+
+        removed_domain_ids = self._parse_domain_ids(removed_domains, "removed domains")
+        if removed_domain_ids is None:
+            return redirect(reverse("invitedmember-domains", kwargs={"pk": pk}))
+
+        if added_domain_ids or removed_domain_ids:
+            try:
+                self._process_added_domains(added_domain_ids, email)
+                self._process_removed_domains(removed_domain_ids, email)
+                messages.success(request, "The domain assignment changes have been saved.")
+                return redirect(reverse("invitedmember-domains", kwargs={"pk": pk}))
+            except IntegrityError:
+                messages.error(request, "A database error occurred while saving changes.")
+                return redirect(reverse("invitedmember-domains-edit", kwargs={"pk": pk}))
+            except Exception as e:
+                messages.error(request, f"An unexpected error occurred: {str(e)}")
```

Comment:
**change before merging:** please log any error that occurs in the exceptions( this applies to above here as well)

Also let's change this to somewhat match what we have for epp errors where we say to message help@get.gov if the error persists. I'm also a tad unsure about showing the user the error type we get, but I could see that being helpful to help find their issue in the logs.

Reply:
I can log the error type and change the error message as you describe